hasura/graphql-engine
# update track relationship text

### Description


- change format of relationship shown under untracked relationships

### Affected components 


- Console
- Docs

### Related Issues




### Solution and Design



### Steps to test and verify


Create author - article tables with a foerign-key and check the Data landing page untracked relationships.

### Limitations, known bugs & workarounds
